h5bp/web_performance/compression.conf

```diff
@@ -49,6 +50,7 @@ gzip_types
   application/x-web-app-manifest+json
   application/xhtml+xml
   application/xml
+  application/xml+rss
```

Comment:
That one is very uncommon. Are you sure of it?

Reply:
Not sure how much of audience is using this.

Reply:
Well, a simple [Google search](https://www.google.com/search?q=application%2Fxml%2Brss) only return `application/rss+xml` which is much more common.

Reply:
Okay, Thanks have removed this.